metrics/README.md

```diff
@@ -0,0 +1,22 @@
+metrics
+=======
+
+The metrics package provides a simple metrics "Reporter" interface which allows the user to submit counters, gauges and timings (along with key/value tags).
+
+Implementations
+---------------
+
+* Prometheus (pull): will be first
+* Prometheus (push): certainly achievable
+* InfluxDB: could quite easily be done
```

Comment:
For now lets stick with "memory" and "prometheus". If the interface has to change we want to make our lives easier.

Reply:
Yes, agreed. I'm really just showing that this doesn't have to be Prometheus-centric here.